_includes/calendar.html

```diff
@@ -35,8 +35,8 @@ <h2 class="page-header" id="calendar">Calendar</h2>
       </td>
       <td>
               {% if item.lab %}
-        <a href="labs/{{ item.lab }}">
-          Lab {{ item.lab }}
+        <a href="labs/{{ item.id }}">
```

Comment:
Couldn't this just be `<a href="{{ item.lab }}">`? Doesn't really matter much though, I think this works fine.

Reply:
No because `item.lab` might be a URL itself, and we don't want it to look like `<a href="labs/http://blahblahblah">`

Reply:
Did you read my comment fully? I'm suggesting just using the URL directly in the `href` attribute instead of using an extra redirect here essentially.

Reply:
Sorry, yeah I misread. I agree that this should eventually be changed to get rid of the redirect. But we would still have to figure out how to handle links to normal lab pages. I don't think those should go in `materials.yaml`